### ISSUE TITLE
Build fails out of the box

### ISSUE THREAD

**Issue by kittylyst** [ebullient/monster-combat#140]
```
./mvnw install
```

fails with:

```
Pulling quarkus-micrometer (ebullient/mc-quarkus-micrometer:latest-jvm)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling quarkus-micrometer (ebullient/mc-quarkus-micrometer:latest-jvm)...
ERROR: pull access denied for ebullient/mc-quarkus-micrometer, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

On Linux Fedora 34, wioth actual Docker (not my podman experiments)

**Comment by ebullient** [comment]
that's weird. I don't publish images for these. It should always be recreated?

**Comment by ebullient** [comment]
Should have been fixed by #222
